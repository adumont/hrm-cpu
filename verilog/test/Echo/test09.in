14 a9
41 28
41 0d
42 da
05 ab
45 0e
05 a4
15 18
56 a6
10 d9
19 6d
15 97
01 de
33 9a
40 8e
45 c9
29 bd
20 90
28 af
10 2a
28 48
59 6d
19 85
14 4f
57 13
