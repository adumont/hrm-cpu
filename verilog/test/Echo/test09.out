a9
28
0d
da
ab
0e
a4
18
a6
d9
6d
97
de
9a
8e
c9
bd
90
af
2a
48
6d
85
4f
13
